Source/Evolve/WarpXEvolve.cpp

```diff
@@ -246,7 +246,12 @@ WarpX::Evolve (int numsteps)
                     // This is currently a lab frame calculation.
                     ComputeMagnetostaticField();
                 }
-                AddExternalFields();
+                // Since the fields were reset above, the external fields are added
+                // back on to the fine patch fields. This make it so that the net fields
+                // are the composite of the field solution and any external field.
```

Comment:
Would `sum` be a less ambiguous term?
```suggestion
                // are the sum of the field solution and any external field.
```

Reply:
Sure :)